pybio/spec/schema.py

```diff
@@ -0,0 +1,253 @@
+from dataclasses import asdict
+from typing import Any, Dict, Union
+
+from marshmallow import Schema as _Schema, pprint, ValidationError, post_load, validates_schema, validates
+
+from pybio.spec import pybio_types, fields
+from pybio.spec.pybio_types import MagicTensorsValue, MagicShapeValue
+
+
+class Schema(_Schema):
+    @post_load
```

Comment:
If I understand this correctly, the `post_load` decorator will automatically return the corresponding dataclass
when deserializing a `Schema`. Do we want that? Having access to the `Schema` would allow to use all the fancy `dump()` functionality that marshmallow provides directly.
We could just leave this as a normal method that returns the dataclass.